### PR TITLE
Fix #2547, upgrade Jetty test dep (only relevant wrt SGv1 endpoints)

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Updates Jetty-core dependency (used by DropWizard, used by SGv1 endpoints)

**Which issue(s) this PR fixes**:

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: coordinator/pom.xml [hunk 1]
```diff
@@ -63,9 +63,9 @@
     <dropwizard.metrics.version>4.1.36</dropwizard.metrics.version>
     <!--
       Note that this is currently aligned with the Jetty version that DropWizard transitively depends on.
-      It's probably a good idea to keep the two versions in sync.
+      It's probably a good idea to keep the two versions in sync (or close)
     -->
-    <jetty.version>9.4.50.v20221201</jetty.version>
+    <jetty.version>9.4.51.v20230217</jetty.version>
```
Comment (ivansenic) [inline coordinator/pom.xml:68]: I would actually also like to see Dropwizard being bumped to `2.1.16`, which actually uses this version.. That being said, we could then invest time in bumping the dependencies across the project.. For example, logback could be bumped one version, maybe something else as well..

Comment (tatu-at-datastax) [reply coordinator/pom.xml:68]: I can try to see if that works fine, sure.

Comment (tatu-at-datastax) [reply coordinator/pom.xml:68]: @ivansenic Unfortunately it's not plug-in replacement without changes. I could spend more time figuring it out, but we are trying to get rid of DropWizard so for this change there's not much benefit.

**Review by ivansenic — APPROVED** [review]

File: coordinator/pom.xml (hunk 1 above)
Comment (tatu-at-datastax) [reply coordinator/pom.xml:68]: Ok yes I will file an issue.

**Pull request merged**